examples/admins/main.tf

```diff
@@ -0,0 +1,33 @@
+module "naming" {
```

Comment:
This example is not really needed in my opinion. Should be a property with defaults just like the others.

Reply:
@jordyvpaassen I am a bit hesitant if we should include the admins feature in de default example.